policy/runtime/runtime.go

```diff
@@ -86,7 +86,7 @@ func NewTemplate(res model.Resolver,
 				"evaluator production limit set to %d, but %d found",
 				t.limits.EvaluatorProductionLimit, prodCnt)
 		}
-		eval, err := t.newEvaluator(mdl.Evaluator, t.exprOpts...)
+		eval, err := t.newEvaluator(mdl.Evaluator, t.limits.EvaluatorExprCostLimit, t.exprOpts...)
```

Comment:
Since the template limits are accessible within the `newEvaluator` call, do you need to provide this argument here? All other limits are accessed directly from `t.limits` within the `newEvaluator` call.

Reply:
I added it so that the limit is only applied to the evaluator. Do we want it to be applicable to the validator as well?

Reply:
Ah. Good point. No, this is fine.